Add open rover page

shoemakerlevy9 opened the pull request:
Hi,

This is Nick from Rover Robotics. I have added a page for our Open Rover product. I tested locally on Chrome, Ubuntu 18.04.

Thanks,
Nick

tfoote commented on _posts/2019-02-11-open-rover.md:
```diff
@@ -0,0 +1,29 @@
+- 'mobile base'
+twitter_text: Checkout the Open Rover
+website: https://roverrobotics.com/
+wiki_homepage: http://wiki.ros.org/rover_robotics
```
Please move this page to http://wiki.ros.org/Robot/rover_robotics and add some content to it. In particular it should at least point to the wiki pages of the driver packages. ala http://wiki.ros.org/rr_openrover_basic

shoemakerlevy9 replied:
see comment below

shoemakerlevy9 commented:
I have changed Wiki homepage from http://wiki.ros.org/rover_robotics to http://wiki.ros.org/Robots/rover_robotics. How do I delete http://wiki.ros.org/rover_robotics?

I have changed the post to point at the new wiki homepage. I also added a Video and a link to the wiki for our ROS driver.

Let me know if there is anything else you want me to do before accepting this PR.

tfoote commented:
> How do I delete http://wiki.ros.org/rover_robotics?

There's a drop down menu on the right side of the page with delete as an option.
